### DISCUSSION TITLE
Les énigmes

### DISCUSSION THREAD

**Discussion by evefevrier** [integration2/quizTIM2023#29] [Q&A] [answered]
Commentaires sur ce quiz

**Accès**
- Serveur de développement (protégé) : https://timunix3.csfoy.ca/~quiz-TIM/alexandradias/index.html
- Dépôt Github (privé pour le moment) : https://github.com/alexandradias/quiz-tim
- Github.io ?

**Comment by evefevrier** [comment, marked as answer]
@alexandradias 
Bravo, tu as réussi à finir ton quiz! Que de défis d'intégration dans ta conception graphique!
Patience... d'ici peu tu seras capable de coder tes brillantes idées.
Petite faute de français dans l'intro et qui est répétée dans plusieurs questions: 
"Seriez-vous reconnaitre les pays qui y sont liés?" devrait s'écrire
**"Sauriez-vous..."**

**Reply by alexandradias** [reply]
Mon dieu!!!!! quelle horrible erreur…
Merci à toi…
